sage-ios/SAGE/SAGE/SignUpRequestTableViewCell.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  CheckinRequestTableViewCell.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 11/14/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import UIKit
+import FontAwesomeKit
+
+class SignUpRequestTableViewCell: UITableViewCell {
+    
+    var userPicture = UIImageView()
+    var userName = UILabel()
+    var schoolAndHours = UILabel()
+    var checkButton = UIButton()
+    var xButton = UIButton()
+    
+    var userID: Int?
+    
+    init() {
+        super.init(style: .Default, reuseIdentifier: "SignUpRequestCell")
```

Comment:
you should override init(style:reuseIdentifier) you shouldnt manually set a reuse identifier here. you should set it in the controller